docs/structure/index.md

```diff
@@ -0,0 +1,27 @@
+# Community structure
+
+The community is currently entirely volunteer-driven, with no formally allocated time or resources for anyone.
+As the community grows and develops, we may need to think about more robust infrastructure concerning its operation and governance.
+
+Currently, there are two main focuses of the community:
+
+1. Quarterly community events
+2. Community Working Groups
+
+## Quarterly Community Events
```

Comment:
```suggestion
## Quarterly Community Events

```
I think you need a space here:
![image](https://github.com/rse-tre/community/assets/1644105/9c02049c-0572-4a72-904a-ab0c470f109b)